sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -1,7 +1,19 @@
 # Change Log azure-ai-textanalytics
 
-## 1.0.0b5 (Unreleased)
+## 1.0.0b6 (2020-05-27)
```

Comment:
could we add in the changelog that now we are targeting v3.0 instead of 3.0-preview.1 ?

Reply:
good point, going to add that

Reply:
and also in the Readme, kind of like we do in FR https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/formrecognizer/azure-ai-formrecognizer#install-the-package